packages-ts/gauntlet-terra-contracts/src/commands/contracts/multisig/updateMembers.ts

```diff
@@ -0,0 +1,66 @@
+import { CATEGORIES } from '../../../lib/constants'
+import { Member } from '../../../lib/multisig'
+import { isValidAddress } from '../../../lib/utils'
+import { AbstractInstruction, instructionToCommand } from '../../abstract/executionWrapper'
+
+type CommandInput = {
+  add?: string[]
+  remove?: string[]
+}
+
+type ContractInput = {
+  add: Member[]
+  remove: string[]
+}
+
+const makeCommandInput = async (flags: any, args: any[]): Promise<CommandInput> => {
+  return {
+    add: flags.add?.split(','),
```

Comment:
If we default these to an empty array `add: flags.add?.split(',') || []`, the CommandInput could be: 
```js
type CommandInput = {
  add: string[]
  remove: string[]
}
```
So we avoid questioning every time if they exist